src/client/api/restful/faucet_enqueue.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_FAUCET_ENQUEUE_H__
+#define __CLIENT_FAUCET_ENQUEUE_H__
+
+#include <stdint.h>
+
+#include "client/api/restful/response_error.h"
+#include "client/client_service.h"
+#include "core/address.h"
+
+/**
+ * @brief Store address and waiting requests count returned in response of faucet enqueue request
+ *
+ */
+typedef struct {
+  char bech32_address[BECH32_ENCODED_ED25519_ADDRESS_STR_LEN + 1];  ///< The bech32 encoded address that is returned
+                                                                    ///< in response
+  uint64_t waiting_reqs_count;                                      ///< The number of requests in faucet queue
+} req_faucet_tokens_t;
```

Comment:
`req_faucet_tokens_t` is confusng, what is it stand for?
```suggestion
} faucet_enqueue_t;
```

Reply:
Changed to faucet_enqueue_t. I meant `request_faucet_tokens_t`, `faucet_enqueue_t` is better.